scripts/eval/local_data/MODEL_GAUNTLET.md

```diff
@@ -0,0 +1,254 @@
+# Mosaic Model Gauntlet v0
+
+The Mosaic Model Gauntlet is MosaicML’s new technique for evaluating the quality of pretrained foundation models. The Model Gauntlet encompasses 35 different benchmarks collected from a variety of sources, and organized into 6 broad categories of competency that we expect good foundation models to have. We compiled the categories after an extensive review of existing LLM publications, and open source evaluation harnesses such as EleutherAI Eval Harness and Stanford CRFM’s HELM.
```

Comment:
Can we add the radar plot from the blog here near the top?

Reply:
also is this the right directory name / structure for the model gauntlet readme?

Reply:
I feel like we should mention gauntlet in the main eval scripts readme. Even a shorter summary that links to this.